internal/cli/usercmd/push.go

```diff
@@ -202,10 +200,17 @@ func (c *EpinioClient) Push(ctx context.Context, params PushParams) error { // n
 		log.V(3).Info("stage response", "response", stageResponse)
 
 		details.Info("start tailing logs", "StageID", stageResponse.Stage.ID)
-		err = c.stageLogs(details, appRef, stageResponse.Stage.ID)
+		c.stageLogs(appRef, stageResponse.Stage.ID)
 		if err != nil {
 			return err
```

Comment:
Maybe we should Wrap also this `err`.

Reply:
this `err` shouldn't exist. The `stageLogs` method no longer returns an error. It's running in a separate go routine and simply prints the error if it occurs. I'm removing this `if`.